src/UI/Components/NextButton.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
```

Comment:
Is there anything special about next button? If not, you can have generic button which accepts text and width or something like that. Most of the styles should be here for the button.

If there is need to give styles at the spot or pass them as parameters then this component would fulfill the purpose of reusable button.